### PR TITLE
deprecating vparquet v1

### PR DESCRIPTION
**Pull request by ie-pham**
**What this PR does**: Deprecating vParquet v1. vParquet v1 blocks can still be read but write is no longer allowed. 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [x] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/configuration/parquet.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ You can still use the previous format `vParquet2`.
 To enable it, set the block version option to `vParquet2` in the Storage section of the configuration file.
```
Comment (joe-elliott) [inline docs/sources/tempo/configuration/parquet.md:32]: this is weird advice. we should likely not be advising anyone to set their version to vParquet2 at this point.

File: tempodb/encoding/common/config.go [hunk 2]
```diff
@@ -65,5 +65,9 @@ func ValidateConfig(b *BlockConfig) error {
 		return fmt.Errorf("positive value required for bloom-filter shard size")
 	}
 
+	if b.Version == "vParquet" {
```
Comment (joe-elliott) [inline tempodb/encoding/common/config.go:68]: can be replaced with this constant:

https://github.com/grafana/tempo/blob/main/tempodb/encoding/vparquet/encoding.go#L14

File: tempodb/encoding/versioned.go [hunk 3]
```diff
@@ -85,7 +85,7 @@ func LatestEncoding() VersionedEncoding {
 func AllEncodings() []VersionedEncoding {
 	return []VersionedEncoding{
 		v2.Encoding{},
-		vparquet.Encoding{},
+		//	vparquet.Encoding{},
```
Comment (joe-elliott) [inline tempodb/encoding/versioned.go:88]: this will help with tests. good call.

**Review by joe-elliott — APPROVED** [review]

File: tempodb/encoding/common/config.go (hunk 2 above)
Comment (joe-elliott) [reply tempodb/encoding/common/config.go:68]: ah, i'm good with this then 👍

File: docs/sources/tempo/configuration/parquet.md (hunk 1 above)
Comment (ie-pham) [reply docs/sources/tempo/configuration/parquet.md:32]: Well, the full context was. Default = vParquet3. But "You can still use the previous format `vParquet2`."

File: tempodb/encoding/versioned.go (hunk 3 above)
Comment (electron0zero) [reply tempodb/encoding/versioned.go:88]: can we remove it instead of commenting it out?

**Review by electron0zero — APPROVED** [review]
one nit, rest lgtm. 

let's goooo 🚀

**Pull request merged**